### PR TITLE
build: upgrade fuels to 0.95.0

### PR DESCRIPTION
**Pull request by maschad**
## Summary
Updates all `fuels` versions to the the latest verisons as well as  `@fuels/connectors` and `@fuels/react`, removes deprecated  `@fuel-wallet/sdk` , `@fuel-wallet/types` as they have been consolidated into `fuels`

```
LICENSE:
The Fuel Support team submits this PR to help you use the fuel SDKs. It is provided "as is" 
and without warranty for any purpose. We are not participants in the project at hand. 
All the rights of this code are reserved to the authors of this repository.
```

### CODE REVIEW

**Review by arboleya — APPROVED** [review]
🧹 🎯

File: package.json [hunk 1]
```diff
@@ -15,24 +15,21 @@
     "@chakra-ui/react": "2.8.0",
     "@emotion/react": "11.11.1",
     "@emotion/styled": "11.11.0",
-    "@fuel-ts/providers": "^0.73.0",
-    "@fuel-wallet/sdk": "0.16.1",
-    "@fuel-wallet/types": "0.15.2",
-    "@fuels/connectors": "^0.2.2",
-    "@fuels/react": "0.18.1",
+    "@fuels/connectors": "^0.31.3",
+    "@fuels/react": "0.31.3",
```
Comment (petertonysmith94) [inline package.json:19]:
```suggestion
    "@fuels/connectors": "^0.32.0",
    "@fuels/react": "0.32.0",
```

**Review by arboleya — APPROVED** [review]

**Comment by petertonysmith94** [comment]
Superseded by #505

**Comment by petertonysmith94** [comment]
@maschad - please close as this is superseded by #505

**Pull request closed without merging**